src/The49.Maui.BottomSheet.csproj

```diff
@@ -1,25 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net7.0;net7.0-android;net7.0-ios;net7.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);net7.0-windows10.0.19041.0</TargetFrameworks>
-		<!-- Uncomment to also build the tizen app. You will need to install tizen by following this: https://github.com/Samsung/Tizen.NET -->
-		<!-- <TargetFrameworks>$(TargetFrameworks);net7.0-tizen</TargetFrameworks> -->
+		<TargetFrameworks>net8.0-android;net8.0-ios</TargetFrameworks>

```

Comment:
By removing the net7 target, apps still on net7 cannot get the new version. I don't think this is a problem because in a couple of months net7 is out of support. Also by removing the desktop targets the consumers of this package need to make the package optional for these targets.

Reply:
@desjoerd: How would I make this package optional for the desktop targets?